src/pages/Registration.css

```diff
@@ -0,0 +1,26 @@
+.divFormRegister {
+    display: flex;
+    flex-direction: column;
+    padding: 6px;
+};
+
+.register-title {
+    display: flex;
+    justify-content: center;
+}
+
+.register-input {
+    padding: 4px;
+    margin: 4px;
+    border-radius: 6px;
+}
+
+.register-btn {
+    padding: 4px;
+    border-radius: 6px;
+}
+
+.register-notification {
+    color: red;
+    font-size: 14px;
+}
```

Comment:
doesn't matter how to call class camelCase or kebab-case. But discuss this question with team and choose how you will do it and leave only one name style in all css files

Reply:
corrected